data/json/techniques.json

```diff
@@ -1070,5 +1070,68 @@
     "knockback_dist": 1,
     "stun_dur": 2,
     "messages": [ "You brutally skewer %s", "<npcname> brutally skewers %s" ]
+  },
+  {
+    "type": "technique",
+    "id": "tec_wingchun_chainpunch",
+    "name": "Chain Punch",
+    "min_unarmed": 1,
+    "unarmed_allowed": true,
+    "strictly_unarmed": true,
+    "knockback_dist": 1,
+    "knockback_spread": 1,
+    "knockback_follow": 1,
+    "mult_bonuses": [ [ "movecost", 0.5 ], [ "damage", "bash", 0.66 ], [ "damage", "cut", 0.66 ], [ "damage", "stab", 0.66 ] ],
+    "messages": [ "You chain strike %s", "<npcname> chain strikes %s" ],
+    "description": "50% moves, 66% damage, knockback and follow"
+  },
+  {
+    "type": "technique",
+    "id": "tec_wingchun_break",
+    "name": "Grab Break",
+    "unarmed_allowed": true,
+    "melee_allowed": true,
+    "defensive": true,
+    "grab_break": true
+  },
```

Comment:
Please add descriptions to all your new techniques.

Reply:
Done.